src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './variables';
+@import './header';
+@import './catalog';
+@import 'card';
+@import 'stars';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: $font-family;
```

Comment:
Ensure that the variable `$font-family` is defined in your SCSS files. If it's not defined, this will cause a compilation error.